src/components/dashboard-page/storage-dashboard-odf/status-card/utils.tsx

```diff
@@ -0,0 +1,53 @@
+import * as _ from 'lodash';
+//import { WatchFlashSystemResource } from '../../../../types';
+import { StorageInstanceKind } from '../../../../types';
+//import { IBM_FlASHSYSTEM } from '../../../../constants/index';
+
+enum HealthState {
+  OK = "OK",
+  ERROR = "ERROR",
+  WARNING = "WARNING",
+  LOADING = "LOADING",
+  UNKNOWN = "UNKNOWN",
+  UPDATING = "UPDATING",
+  PROGRESS = "PROGRESS",
+  NOT_AVAILABLE = "NOT_AVAILABLE",
+}
+
+const FlashsystemHealthStatus = {
+  Ready: {
+    state: HealthState.OK,
+  },
+  'Not Ready': {
+    state: HealthState.WARNING,
+    message: 'Warning',
+  },
+  Error: {
+    state: HealthState.ERROR,
+    message: 'Error',
+  },
+  PROCESSING: {
```

Comment:
Phases are always in camel case in general. 
Just pointing in case its not a miss.